spec/models/like_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  let(:user) { User.create(name: 'Tom', photo_url: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.') }
+  let(:post) { Post.create(author: user, title: 'Mission Impossible', text: 'Fallout', comments_counter: 0) }
+  let(:like) { Like.create(user:, post:) }
+
+  it 'should show an error if likes_counter is negative' do
+    post.likes_counter = -2
+    expect(post).not_to be_valid
+  end
+
+  it 'should be valid when likes_counter is zero' do
+    post.likes_counter = 0
+    expect(post).to be_valid
+  end
+
+  it 'should be valid when likes_counter is positive' do
+    post.likes_counter = 2
+    expect(post).to be_valid
+  end
+end
```

Comment:
- [ ] Great job adding the validations and testing them 👏. However, as stated in the requirement, all your models' methods should be tested too. So please, I would suggest you test the `update_likes_counter` method.

### Requirement :point_down: 
![Screenshot from 2023-12-15 16-53-41](https://github.com/Surafels/Blog-app/assets/89228743/2e79dda6-b073-458d-8fd9-45bc19a9024c)